src/views/DataExploration/components/SetsManagementDropdown/index.tsx

```diff
@@ -88,6 +88,9 @@ const itemIcon = (type: string) => {
   }
 };
 
+const singularizeFilesSetType = (type: string) =>
+  type === SetType.FILES ? type.slice(0, -1) : type;
+
```

Comment:
Hacky, but we rely on `SetType` enum for the wording and did not know how to handle this issue differently without changing a lot of code.